src/Events.ts

```diff
@@ -1,12 +1,14 @@
 export type CoveoEnvironment = "dev" | "stg" | "prd";
 export const isCoveoEnvironment = (value: unknown): value is CoveoEnvironment =>
-    typeof value === 'string' && /dev|stg|prod/.test(value);
+    typeof value === 'string' && /dev|stg|prd/.test(value);
```

Comment:
I guess this just worked accidentally since it defaulted to `prd` if the environment wasn't recognized :smile: